client-generated/models/GetApiKeysResponseDto.ts

```diff
@@ -0,0 +1,24 @@
+// tslint:disable
+/**
+ * msdoc server API
+ * The msdoc server API description
+ *
+ * The version of the OpenAPI document: 1.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+/**
+ * @export
+ * @interface GetApiKeysResponseDto
+ */
+export interface GetApiKeysResponseDto {
+    /**
+     * @type {Array<string>}
+     * @memberof GetApiKeysResponseDto
+     */
+    apiKeys: Array<string>;
+}
```

Comment:
This type does not look correct to me. When loading the API keys using `GET /api-keys`, I am getting an array of objects, not an array of strings. These objects contain the key ID, key name, and the creation timestamp. Or is `GetApiKeysResponseDto` used on a different endpoint?

Reply:
Also, I noticed that the returned objects do not contain the keys themselves. Is this indended?

Reply:
Very good catch, thx!

Reply:
> Also, I noticed that the returned objects do not contain the keys themselves. Is this indended?

Yes, that is intended. The key is only visible on creation. I think this mimics also how GitHub and other platforms handle API keys creation.